ckanext/saml2/plugin.py

```diff
@@ -213,6 +258,7 @@ def before_map(self, map):
                       action='saml2_unauthorized')
             m.connect('saml2_slo', '/slo', action='slo')
             m.connect('staff_login', '/service/login', action='staff_login')
+            m.connect('saml2_user_delete', '/user/delete/{id}', action='delete')
```

Comment:
If you're overriding the user_delete action at line 615 why map a custom controller here?